models/business-unit/src/types.ts

```diff
@@ -17,12 +17,8 @@ export type TDivisionDraftBuilder = TBuilder<TDivisionDraft>;
 export type TCreateCompanyDraftBuilder = () => TCompanyDraftBuilder;
 export type TCreateDivisionDraftBuilder = () => TDivisionDraftBuilder;
 //BusinessUnitDraftGraphql is only scaffolding at this time
-export type TCompanyDraftGraphql = TCompanyDraft & {
-  __typename: 'BusinessUnitDraft';
```

Comment:
For most of the models in this PR (save for `Reference`, `KeyReference`, and `LocalizedString`), this was the relevant change. Remove `__typename` from `types.ts`, the generator, and ensure the type was safe all the way down to tests